newsletter-july-2021.rst

```diff
@@ -0,0 +1,118 @@
+.. post:: July 8, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from July.
+
+Read the Docs newsletter - July 2021
+====================================
+
+Welcome to a new edition of our monthly newsletter, where we
+openly share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- Eric_ presented the history of the company at the Upstream 2021 conference
+  along with ideas for open source sustainability.
+  You can read more about it in :doc:`our recent blog post </behind-the-scenes-2021-1>`.
+- We revamped our `sustainability page <https://readthedocs.org/sustainability/>`_
+  to clarify the ways in which you or your project can go ad-free.
+- The first two parts of the Sphinx tutorial we are writing
+  in the context of the :doc:`CZI grant </czi-grant-announcement>`
+  were merged to the development branch,
+  and will be publicly available soon.
+- We finalized the design of our upcoming
+  `Embed API v3 <https://docs.readthedocs.io/en/stable/development/design/embed-api.html>`_,
+  which will allow sites outside Read the Docs to embed our documentation.
```

Comment:
```suggestion
  which will allow embedding sites hosted outside Read the Docs in your documentation.
```

Reply:
I think I understood this backwards, asked for clarification online

Reply:
Turns out that there will be a plethora of new use cases :) Will write something more generic